CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,135 @@
+community a harassment-free experience for everyone, regardless of age, body
+size, visible or invisible disability, ethnicity, sex characteristics, gender
+identity and expression, level of experience, education, socio-economic status,
+nationality, personal appearance, race, caste, color, religion, or sexual
+identity and orientation.
+
+We pledge to act and interact in ways that contribute to an open, welcoming,
+diverse, inclusive, and healthy community.
+
+## Our Standards
+
+Examples of behavior that contributes to a positive environment for our
+community include:
+
+* Demonstrating empathy and kindness toward other people
+* Being respectful of differing opinions, viewpoints, and experiences
+* Giving and gracefully accepting constructive feedback
+* Accepting responsibility and apologizing to those affected by our mistakes,
+  and learning from the experience
+* Focusing on what is best not just for us as individuals, but for the overall
+  community
+
+Examples of unacceptable behavior include:
+
+* The use of sexualized language or imagery, and sexual attention or advances of
+  any kind
+* Trolling, insulting or derogatory comments, and personal or political attacks
+* Public or private harassment
+* Publishing others' private information, such as a physical or email address,
+  without their explicit permission
+* Other conduct which could reasonably be considered inappropriate in a
+  professional setting
+
+## Enforcement Responsibilities
+
+Community leaders are responsible for clarifying and enforcing our standards of
+acceptable behavior and will take appropriate and fair corrective action in
+response to any behavior that they deem inappropriate, threatening, offensive,
+or harmful.
+
+Community leaders have the right and responsibility to remove, edit, or reject
+comments, commits, code, wiki edits, issues, and other contributions that are
+not aligned to this Code of Conduct, and will communicate reasons for moderation
+decisions when appropriate.
+
+## Scope
+
+This Code of Conduct applies within all community spaces, and also applies when
+an individual is representing the community in public spaces.
+
+Our community consists of contributors in their efforts related to the development and distribution of any materials for the HoloViz organization. The projects in this community include the repositories in the [HoloViz GitHub organization](https://github.com/holoviz), such as Param, HoloViews, hvPlot, Panel, GeoViews, Lumen, Datashader, and Colorcet. Other efforts, whether online or in-person, undertaken by this community and coordinated through [HoloViz communication channels](https://holoviz.org/community.html) are also covered under this Code of Conduct. This code applies equally to founders, developers, mentors and new community members, in all spaces managed by HoloViz. In addition, violations of this code outside these spaces may affect a person’s ability to participate within them.
```

Comment:
The project order seems arbitrary, perhaps it should be sorted in alphabetic order?